### PR TITLE
Normalize handlebars comments to {{!-- syntax

### PR DESCRIPTION
**Pull request by locks**
Updating documentation as mentioned in https://github.com/ember-template-lint/ember-template-lint/pull/1011#issuecomment-565607953.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,13 +213,13 @@ An in-element instruction with the `-tree` suffix will apply to that element and
...
   <hr>
-  <p {{! template-lint-configure-tree block-indentation "tab" }}>
+  <p {{!-- template-lint-configure-tree block-indentation "tab"  --}}>
     <span>Hello!</span>
   </p>
 </div>
 ```
 
-Note that enabling a rule (`{{! template-lint-enable }}`) that has been configured in-template (`{{! template-lint-configure }}`), will restore it to its default configuration rather than the modified in-template configuration for the scope of the `{{! template-lint-enable }}` instruction.
+Note that enabling a rule (`{{!-- template-lint-enable }}`) that has been configured in-template (`{{!-- template-lint-configure }}`), will restore it to its default configuration rather than the modified in-template configuration for the scope of the `{{!-- template-lint-enable  -- -- --}}` instruction.
```
Comment (bmish) [inline README.md]: The conversion has mistakes in this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -909,19 +909,19 @@ You can read more about the rule (and configuration) [in the documentation](http
...
 
   A few new types of control statements are now available:
-    * `{{! template-lint-enable some-rule-name }}` - This will enable the rule `some-rule-name` with the default configuration (from `.template-lintrc.js`) or `true` (if not present in the config file). This can be ran for multiple rules at once (i.e. `{{! template-lint-enable bare-strings some-other-thing }}`).
-    * `{{! template-lint-disable some-rule-name }}` - This will disable the rule `some-rule-name`. Multiple rules can be provided at once (i.e. `{{! template-lint-disable bare-strings some-other-thing }}`).
-    * `{{! template-lint-configure some-rule-name { "whitelist": ["some", "valid", "json"] } }}` - This configures the rule `some-rule-name` with the `JSON.parse()`'ed result of the second argument.  The configure instruction only applies toa  single rule at a time.
+    * `{{!-- template-lint-enable some-rule-name }}` - This will enable the rule `some-rule-name` with the default configuration (from `.template-lintrc.js`) or `true` (if not present in the config file). This can be ran for multiple rules at once (i.e. `{{! template-lint-enable bare-strings some-other-thing  --}}`).
```
Comment (bmish) [inline CHANGELOG.md]: Missing `--}}` on part of these lines.

Comment (locks) [reply CHANGELOG.md]: Darn, the CHANGELOG shouldn't have been included at all. Updating!

**Comment by locks** [comment]
@bmish thank you, comments should be addressed!

**Pull request merged**